docs/pages/reference/access-controls/access-monitoring-rules.mdx

```diff
@@ -208,6 +234,12 @@ condition: set("role_1", "role_2").contains_all(access_request.spec.roles)
 
 # Applies if the user has trait "team: dev" or "team: stage".
 condition: contains_any(user.traits["team"], set("dev". "stage"))
+
+# Applies if all requested resources have the `env: dev` label.
+condition: access_request.spec.resource_labels_intersection["env"].contains("dev")
+
+# Applies if any requested resources has the `env: prod` label.
+condition: access_request.spec.resource_labels_union["env"].contains("prod")
```

Comment:
Is there even a real use-case for when the labels union is useful? Maybe we should consider removing it for now (before it's "officially" shipped), otherwise it seems like an easy way to shoot yourself in the foot and create a too overzealous rule.

Reply:
The labels union allows users to write rules where there exists at least one resource with a specific label. I think this can come in handy for notification routing, or for automatically denying requests. 

Example: admins might want to be notified if any user requests access to a `prod` resource.
Example: admins might want to deny any access requests for a `prod` resource from interns.